addons/hr/models/hr_employee.py

```diff
@@ -148,6 +148,16 @@ def _lang_get(self):
     study_school = fields.Char("School", groups="hr.group_hr_user", tracking=True)
     emergency_contact = fields.Char(groups="hr.group_hr_user", tracking=True)
     emergency_phone = fields.Char(groups="hr.group_hr_user", tracking=True)
+    contract_date_start = fields.Date(readonly=False, related="version_id.contract_date_start", inherited=True, groups="hr.group_hr_manager")
```

Comment:
Maybe all this isn't necessary anymore since you updated `_prepare_create_values`?

Reply:
maybe I will check again we will see